lib/src/network/protocol/http.rs

```diff
@@ -481,35 +497,40 @@ impl<Front:SocketHandler> Http<Front> {
       return ClientResult::Continue;
     }
 
+    
+
     match unwrap_msg!(self.state.as_ref()).response {
       // FIXME: should only restart parsing if we are using keepalive
-      Some(ResponseState::Response(_,_))                            |
-      Some(ResponseState::ResponseWithBody(_,_,_))                  |
-      Some(ResponseState::ResponseWithBodyChunks(_,_,Chunk::Ended)) => {
+      Some(ResponseState::Response(_, _))                            |
```

Comment:
please refrain from making cosmetics changes that are unnecessary for the commit